backend/game/turn_actions/heal.py

```diff
@@ -0,0 +1,7 @@
+import game.player.player as player
+import game.values.locations as locations
+
+
+def heal_self(player_healing: player.Player, health_points_to_add):
+    if player_healing.location is not locations.Locations.TOKYO:
+        player_healing.update_health_by(health_points_to_add)
```

Comment:
So we need to be careful here.
The rule is: the player cannot heal _from dice_ while in Tokyo. 
The player _can_ heal from cards or other means while in Tokyo.
We need test cases for all of this as well.

Reply:
My thought was this would only be used from dice resolution (should have named it better) since cards seemed to be using the player.update_health_by() method. That was why I left alone the card part. Turns out the heal card I was referencing wasn't in master but I saw it cause I accidentally made a branch the same name as @otaiwo70 who was working on the heal card. To better reflect my original intention heal_self should be renamed to heal_self_from_dice. But if we're changing the heal card then I need to account for that.